docs/metrics/README.md

```diff
@@ -405,7 +405,7 @@ the first time an overflow is detected for a given metric.
 When [Delta Aggregation
 Temporality](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/data-model.md#temporality)
 is used, it is possible to choose a smaller cardinality limit by allowing the
-SDK to reclaim unused metric points.
+SDK to reclaim unused combinations.
```

Comment:
I think we should revert it back, two lines below "metric points" is used, and the environment variable is also mentioning "METRIC_POINT". Besides, metric point is covered by the spec https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/data-model.md#metric-points.

Reply:
I reverted the change and tried to explain how cardinality limit relates to `MetricPoint`s: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5349/commits/2367fe6f11159e6281751e7069c15c6bb5f11e1a